model/src/commonMain/kotlin/repositories/OrtRunRepository.kt

```diff
@@ -57,6 +57,13 @@ interface OrtRunRepository {
      */
     fun getByIndex(repositoryId: Long, ortRunIndex: Long): OrtRun?
 
+    /**
+     * Gets the id of an ORT run by its [index][ortRunIndex] within a [repository][repositoryId].
```

Comment:
Please use imperative voice: "Get the id...". I would not use the term "lightweight" to describe this operation; it just serves a different use case. If a client actually needs the `OrtRun` and only has the repository ID and index, there is even an additional step required.